ts/components/screens/ListItemComponent.tsx

```diff
@@ -73,65 +77,70 @@ const styles = StyleSheet.create({
 export default class ListItemComponent extends React.Component<Props> {
   public render() {
     return (
-      <ListItem
-        style={[styles.listItem, styles.flexRow, this.props.style]}
+      <TouchableWithoutOpacity
+        style={styles.highlight}
         onPress={this.props.onPress}
         onLongPress={this.props.onLongPress}
-        first={this.props.isFirstItem}
-        last={this.props.isLastItem || this.props.hideSeparator}
       >
-        <View style={styles.flexColumn}>
-          <View style={styles.flexRow}>
-            <View style={styles.flexRow2}>
-              {this.props.hasBadge && (
-                <View style={styles.spacingBase}>
-                  <BadgeComponent />
-                </View>
-              )}
+        <ListItem
```

Comment:
This looks like an hacking and not a clean solution.
Try to revert to the original and put *underlayColor* prop inside **ListItem** to have the wanted effect
(maybe _underlayColor_ isn't in type file definition. If so you could make a PR to https://github.com/GeekyAnts/NativeBase and a patch for this project)

Reply:
Ok perfect, I made a patch for our project and it works very good :)